composer.json

```
@@ -28,11 +28,11 @@
         "async-aws/event-bridge": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.5",
-        "mnapoli/hard-mode": "^0.2.0",
+        "phpunit/phpunit": "^9.4",
+        "mnapoli/hard-mode": "^0.3.0",
         "phpstan/phpstan": "^0.12.0",
         "symfony/framework-bundle": "^4.3 || ^5.0",
-        "nyholm/symfony-bundle-test": "^1.6",
+        "nyholm/symfony-bundle-test": "dev-master",
```

There is a tagged version now. 

```suggestion
        "nyholm/symfony-bundle-test": "^1.6",
```